### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by jnicolaslaurenti**
Added unit test

_Coverage:_

image

_Tests:_

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# WeatherApp
\ No newline at end of file
+# WeatherApp
```
Comment (SartiMau) [inline README.md:1]: put the dates in the descendent mode pls

File: app/src/main/java/com/onboarding/wheatherapp/mvp/presenter/WeatherAppPresenter.kt [hunk 2]
```diff
@@ -30,7 +30,7 @@ class WeatherAppPresenter(
     }
 
     companion object {
-        private const val CITY = "Tandil"
-        private const val HOUR = "12:00"
+        const val CITY = "Tandil"
```
Comment (SartiMau) [inline app/src/main/java/com/onboarding/wheatherapp/mvp/presenter/WeatherAppPresenter.kt]: why do you let this public?

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/test/java/com/onboarding/wheatherapp/util/WeatherTestUtils.kt [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    const val date_2 = "10/02/2021 17:39:00"
+    const val date_3 = "10/02/2021 12:00:00"
+    val main = Main(TEMP, TEMP_MIN, TEMP_MAX)
+    val weatherDescription = WeatherDescription(ICON_WEATHER_STRING)
+    val listWeather: MutableList<WeatherDescription> = mutableListOf(weatherDescription)
+    val data_1 = Data(main, listWeather, date_1)
+    val data_2 = Data(main, listWeather, date_2)
+    val data_3 = Data(main, listWeather, date_3)
+    val forecastExtendTest: MutableList<Data> = mutableListOf(data_1, data_2, data_3)
+    val expectedFilteredList = listOf<Data>(data_3)
```
Comment (mauroLonghi) [inline app/src/test/java/com/onboarding/wheatherapp/util/WeatherTestUtils.kt]: why you are defining this val`s here?

Comment (mauroLonghi) [reply app/src/test/java/com/onboarding/wheatherapp/util/WeatherTestUtils.kt]: the configuration for each value, must stay in a private method on the test o in each method

File: app/src/main/java/com/onboarding/wheatherapp/mvp/presenter/WeatherAppPresenter.kt (hunk 2 above)
Comment (jnicolaslaurenti) [reply app/src/main/java/com/onboarding/wheatherapp/mvp/presenter/WeatherAppPresenter.kt]: Solved!

File: app/src/test/java/com/onboarding/wheatherapp/util/WeatherTestUtils.kt (hunk 3 above)
Comment (jnicolaslaurenti) [reply app/src/test/java/com/onboarding/wheatherapp/util/WeatherTestUtils.kt]: Solved!

File: README.md (hunk 1 above)
Comment (jnicolaslaurenti) [reply README.md:1]: Solved!

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

**Pull request merged**